### DISCUSSION TITLE
Newbie question about publishing dandisets

### DISCUSSION THREAD

**Discussion by laurelrr** [dandi/helpdesk#133] [General]
Hello,
I am new here and in need of some help in navigating the proper way to get a DOI associated with my dandiset.  In our lab, we typically put together a manuscript for submission, then convert the supporting data to NWB formats and upload to Dandi.  I would like to include a DOI for the dandiset with our initial manuscript submission to which I can then add our data. But I am confused by the draft vs published versions of the dandiset.  Can you help clarify? Ideally, the dandiset would be published in final form before the manuscript is published.  But if there are changes needed to the dandiset, how would this effect the published data link?  Thanks

Note, this is similar to issue #53 but it is unclear to me what people would see if they follow our link to an out of date dandiset.  Is there a way to show that a newer version exists, like on bioRxiv?

**Comment by yarikoptic** [comment]
DRAFT version can still change and we do not assign DOI to them.  When you "publish" a version -- you get a DOI.

if changes needed -- introduce them to DRAFT and publish again to get another new DOI to use in the updated manuscript.

> Is there a way to show that a newer version exists, like on bioRxiv?

well -- it kinda could be seen among list of the versions of the right, but indeed it is not that "obvious". Hence I filed 

- https://github.com/dandi/dandi-archive/issues/1931

Please chime in there if you see how to improve that.
